package.json

```diff
@@ -5,20 +5,40 @@
     "lint": "eslint --ignore-path .gitignore . && stylelint web_modules/**/*.css",
     "eslint-staged": "eslint-staged",
     "stylelint-staged": "stylelint-staged",
-    "start": "babel-node scripts/build --server --dev --open",
-    "static": "babel-node scripts/build --static --production",
+    "start": "cross-env BABEL_ENV=statinamic babel-node scripts/build --server --open --dev",
+    "build": "cross-env BABEL_ENV=statinamic babel-node scripts/build --static --production",
     "pretest": "npm run lint",
-    "test": "npm run static",
-    "posttest": "echo 'postcss.org' > dist/CNAME && touch dist/.nojekyll",
+    "test": "npm run build",
     "styleguide-server": "styleguidist server",
     "styleguide-build": "styleguidist build",
     "predeploy": "npm run build",
     "deploy": "./scripts/deploy.sh"
   },
+  "statinamic": {
+    "CNAME": true
+  },
   "babel": {
-    "stage": 0
+    "presets": [
+      "babel-preset-react",
+      "babel-preset-es2015",
+      "babel-preset-stage-1"
+    ],
+    "env": {
+      "statinamic": {
+        "plugins": [
+          [
+            "babel-plugin-webpack-loaders",
+            {
+              "config": "./scripts/webpack.config.babel.js",
+              "verbose": false
+            }
+          ]
+        ]
+      }
+    }
   },
   "eslintConfig": {
+    "parser": "babel-eslint",
     "extends": [
       "eslint-config-i-am-meticulous",
```

Comment:
you can remove this line, eslint-config-i-am-meticulous@2 /react include the index too.